registry/server/common/services/assets/AssetsManifestReader.ts

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import * as Joi from 'joi';
+import {AssetsManifestError} from './errors/AssetsManifestError';
+
+interface BasicAssetsManifest {
+    spaBundle: string;
+}
+
+export interface SharedLibAssetsManifest extends BasicAssetsManifest {}
+
+export interface ApplicationAssetsManifest extends BasicAssetsManifest {
+    cssBundle?: string;
+    dependencies?: Record<string, string>;
+}
+
+export interface AssetsManifest extends ApplicationAssetsManifest {}
+
+export class AssetsManifestReader {
+    public static async read(assetsManifestUrl: string): Promise<AssetsManifest> {
+        let assetsManifestContent;
+
+        try {
+            const response = await axios.get(assetsManifestUrl, {
+                responseType: 'json',
+            });
+
+            assetsManifestContent = response.data;
+
+        } catch (error) {
+            throw new AssetsManifestError(`"assetsDiscoveryUrl" ${assetsManifestUrl} is not available. Check the url via browser manually.`);
```

Comment:
looks like it hides original error. Can we propagate it?

Reply:
I think in that case we should not distinguish an error. Any network error shows us we have to debug accessibility of asset

Reply:
I understand that, but if we do not have original error in error that is thrown and do not have it in logs, it would be hard to debug